chart/gateway/templates/application.yaml

```diff
@@ -13,32 +13,33 @@ spec:
   descriptor:
     type: Gateway
     version: "{{ .Values.image.tag }}"
+    description: >-
```

Comment:
Are we going to include this generic description in all application charts or is this going to be only in the virtru-public main chart down the road?